src/Microsoft.VisualStudio.Threading/TplExtensions.cs

```diff
@@ -52,7 +52,7 @@ public static void WaitWithoutInlining(this Task task)
             if (!task.IsCompleted)
             {
                 // Waiting on a continuation of a task won't ever inline the predecessor (in .NET 4.x anyway).
-                var continuation = task.ContinueWith(t => { }, CancellationToken.None, TaskContinuationOptions.ExecuteSynchronously, TaskScheduler.Default);
+                var continuation = task.ContinueWith(t => { }, CancellationToken.None, TaskContinuationOptions.None, TaskScheduler.Default);
                 continuation.Wait();
```

Comment:
Interesting inconsistency: Task.Wait() throws if the task failed. Task.WaitWithoutInlining() does not throw.
Is there a reason for that? And should the comment highlight this difference.